src/main/java/tictactoe/BoardConverter.java

```diff
@@ -11,38 +13,87 @@ public String toString(Marks[][] board) {
 
         for (Marks[] columns : board) {
             for (Marks mark : columns) {
-                str.append(createLine(idSpot, mark));
+                str.append(createSpot(idSpot, mark));
```

Comment:
Isn't it StringBuffer which is deprecated instead of StringBuilder?

I agree that in this case StringBuilder it's overkill as the array doesn't contain thousand of line, but if I'm right the concatenation is O(n^2) when the string builder is O(1) as it seems to work like a "stack" which become O(n^4) vs O(n^2) if it's inside a double loop.